Basket-Donation-Payments/Basket-Donation-Payments/Models/Basket.m

```diff
@@ -36,6 +37,7 @@ + (Basket*)initWithName:(NSString*)basketName description:(NSString*)basketDescr
     basket.basketDescription = basketDescription;
     basket.headerPicFile = headerPicFile;
     basket.totalDonatedValue = @0;
+    basket.favoriteCount = 0;
```

Comment:
Same comment from previous PR, storing the favorite count in the basket will cause data consistency issues.

Reply:
Can I fetch immediately before update favorite count so that data stays consistent?

Reply:
This will work for the purpose of this project, just consider it if you ever plan to make this app public, or when you're working on a public app with many users.